src/main/java/com/agilethought/atsceapi/service/UserServiceImpl.java

```diff
@@ -14,7 +15,6 @@
 import com.agilethought.atsceapi.exception.UnauthorizedException;
 import com.agilethought.atsceapi.model.User;
 import com.agilethought.atsceapi.repository.UserRepository;
-import com.agilethought.atsceapi.validator.LoginValidator;

```

Comment:
LoginValidator was removed but I see a dependency in line 34.

Reply:
It's because we inject that dependency to do the post operation for login a user